cmd/utils_test.go

```diff
@@ -232,6 +232,12 @@ func TestLocalAddress(t *testing.T) {
 	if runtime.GOOS == "windows" {
 		return
 	}
+
+	currentIsDistXL := globalIsDistXL
+	defer func() {
+		globalIsDistXL = currentIsDistXL
+	}()
```

Comment:
Pass ``currentIsDistXL`` as an argument to anonymous to force binding of its current value.